src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,37 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import classNames from 'classnames';
+
+type Props = {
+  todo: Todo | null;
+  handlerModButton: (todo: Todo | null) => void;
```

Comment:
Consider renaming `handlerModButton` to `handleModButtonClick` to follow common naming conventions for event handlers and improve code readability.